pkg/cmd/stack/translate.go

```diff
@@ -388,6 +389,54 @@ func translateStatefulSet(svcName string, s *model.Stack) *appsv1.StatefulSet {
 	}
 }
 
+func translateJob(svcName string, s *model.Stack) *batchv1.Job {
+	svc := s.Services[svcName]
+
+	initContainers := getInitContainers(svcName, svc)
+
+	return &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        svcName,
+			Namespace:   s.Namespace,
+			Labels:      translateLabels(svcName, s),
+			Annotations: translateAnnotations(svc),
+		},
+		Spec: batchv1.JobSpec{
+			Parallelism: pointer.Int32Ptr(svc.Replicas),
+			Selector: &metav1.LabelSelector{
+				MatchLabels: translateLabelSelector(svcName, s),
+			},
+			ManualSelector: pointer.BoolPtr(true),
```

Comment:
should we leave this empty?

Reply:
If you leave `ManualSelector` empty you can not add any selector, and when listing the jobs we can not see if it's from another stack

Reply:
From the docs:
```
	// manualSelector controls generation of pod labels and pod selectors.
	// Leave `manualSelector` unset unless you are certain what you are doing.
	// When false or unset, the system pick labels unique to this job
	// and appends those labels to the pod template.  When true,
	// the user is responsible for picking unique labels and specifying
	// the selector.  Failure to pick a unique label may cause this
	// and other jobs to not function correctly.  However, You may see
	// `manualSelector=true` in jobs that were created with the old `extensions/v1beta1`
	// API.
	// More info: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#specifying-your-own-pod-selector
	// +optional
	ManualSelector *bool `json:"manualSelector,omitempty" protobuf:"varint,5,opt,name=manualSelector"`
```
We don't need control on the pod labels, no? we only need control on the job labels to do queries, right?